### DISCUSSION TITLE
Extracting learning rate from MaskedState or ScaleByAdamState

### DISCUSSION THREAD

**Discussion by GeophyAI** [google-deepmind/optax#1072] [General]
How to print the learning rate from a `MaskedState ` or a `ScaleByAdamState` instance? I want to check correctness of the lr schedule, but I failed to find relvent information on website. Could anyone help me?

**Comment by vroulet** [comment]
Did #961 not help you for that?

**Reply by GeophyAI** [reply]
Thank you for your help, it's really what I want. Have a good day, bro!
